compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1155,7 +1155,8 @@ object Parsers {
         t
     }
 
-    def ascription(t: Tree, location: Location.Value) = atPos(startOffset(t), in.skipToken()) {
+    def ascription(t: Tree, location: Location.Value) = atPos(startOffset(t)) {
+      in.skipToken()
```

Comment:
Why the change? The previous implementation would place the `^` under the `:` of an ascription, which seems to be what we want, no?

Reply:
In all other cases, the point is under the first letter of the name of the definition (e.g. for `def foo: Int = 1` the point is under `f`). This change means that I can always use the point of the definition tree to set the symbol position.